Dockerfile

```diff
@@ -24,6 +24,10 @@ RUN cd cmd/sso-proxy/generate-request-signature && go build -mod=readonly -o /bi
 # add static assets and copy binaries from build stage
 # =============================================================================
 FROM debian:stable-slim
+RUN ln -s /usr/bin/dpkg-split /usr/sbin/dpkg-split
+RUN ln -s /usr/bin/dpkg-deb /usr/sbin/dpkg-deb
+RUN ln -s /bin/tar /usr/sbin/tar
+RUN ln -s /bin/rm /usr/sbin/rm
```

Comment:
this was needed for the builds to complete.  seems like the file locations are in different spots on the ARM arch.

Reply:
👍 Would it be worth adding this in a quick inline comment?

Reply:
if this remains, yes.  i want to test the resultant multi-arch images in a arm64 cluster before finalizing this.  i only got to test it once during hackweek before tearing down the temp cluster.

Reply:
i'm going to see if we need this for arm64, which is the primary secondary target we want for M1's and Graviton's

Reply:
that didn't work, these are needed for arm64 also apparently.